### ISSUE TITLE
row.AddCell and row.GetCell has different behavior

### ISSUE THREAD

**Issue by frg-tarasov** [tealeg/xlsx#627]
I create new file and try to fill it.
First I tried to fill with ```row.GetCell``` but file was empty after saving.
If I do it with ```row.AddCell``` then it works correctly.

Below there is full example for reproducing this problem.
```go
package main

import (
	"log"

	"github.com/tealeg/xlsx/v3"
)

func main() {
	xlsxFile1 := xlsx.NewFile()
	xlsxFile2 := xlsx.NewFile()

	sheet1, err := xlsxFile1.AddSheet("sheet1")
	if err != nil {
		log.Fatalf("Error adding sheet1: %v", err)
	}

	sheet2, err := xlsxFile2.AddSheet("sheet2")
	if err != nil {
		log.Fatalf("Error adding sheet2: %v", err)
	}

	sheet1.AddRow().AddCell().SetString("test1")

	sheet2.AddRow().GetCell(0).SetString("test2")

	err = xlsxFile1.Save("./file1.xlsx") // ok - A1 contains "test1"
	if err != nil {
		log.Fatalf("Error saving file1: %v", err) 
	}

	err = xlsxFile2.Save("./file2.xlsx") // wrong - A1 is void
	if err != nil {
		log.Fatalf("Error saving file2: %v", err)
	}
}

```

**Comment by chaganiu** [comment]
Can confirm the same. behavior.  Is there a workaround to insert data in a specific cell?  I tried

```go
cell, _ := worksheet.Cell(row.GetCoordinate(), value)
cell.SetString("foo")
```

but i think that internally calls `GetCell` and so doesn't work

**Comment by frg-tarasov** [comment]
I used this for inserting data by rows. It was useful for my task.

**Comment by Morrowless** [comment]
Same issue with GetCell.

**Comment by tealeg** [comment]
This should be fixed by #642 and therefore in v3.2.2

**Comment by frg-tarasov** [comment]
Thanks!
